generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx.ejs

```diff
@@ -106,7 +106,8 @@ export default class Header extends React.Component<IHeaderProps, IHeaderState>
             <Nav className="ml-auto" navbar>
               <Home />
               {isAuthenticated && <EntitiesMenu />}
-              {isAuthenticated && isAdmin && <AdminMenu showSwagger={isSwaggerEnabled} showDatabase={!isInProduction} />}
+              {isAuthenticated && isAdmin &&
+                <AdminMenu showSwagger={isSwaggerEnabled} <% if (devDatabaseType === 'h2Disk' || devDatabaseType === 'h2Memory') { %> showDatabase={!isInProduction} <%_ } _%> />}
```

Comment:
shouldnt the related code be skipped in the admin menu component as well?

Reply:
they were already, that's what caused the build error in the daily build that @pascalgrimaud mentioned in #7564